mlir/lib/CAPI/Dialect/MIGraphX.cpp

```diff
@@ -57,7 +57,7 @@ void mlirGetKernelInfo(MlirModule module, int *size, void *data) {
     for (int i = 0; i < argSize; i++)
       argData[i + 1] = info[i];
     char *nameData = (char *)(argData + argSize + 1);
-    for (int i = 0; i < kernelName.size(); i++) {
```

Comment:
I suppose size() only returns constant value and compiler can optimize this. good to have the correct type, thanks.